components/Session/Submit/Intro.js

```diff
@@ -21,12 +21,14 @@ const dlog = debug('that:session:intro');
 const CREATE_SESSION = gql`
   mutation createSession($eventId: ID!, $session: SessionCreateInput!) {
     sessions {
-      create(eventId: $eventId, session: $session) {
-        id
-        type
-        title
-        category
-        status
+      create {
```

Comment:
the eventId is here now, not on the child create..

Reply:
@csell5 this is solved per our discussion, correct?